pydicom/values.py

```diff
@@ -192,26 +262,32 @@ def convert_PN(byte_string, is_little_endian, struct_format=None, encoding=None)
 
     # We would like to return string literals
     if not in_py2:
-        valtype = lambda x: PersonName(x, *args).decode()
+        valtype = lambda x: PersonName(x, *args).decode()  # nopep8
     else:
-        valtype = lambda x: PersonName(x, *args)
+        valtype = lambda x: PersonName(x, *args)  # nopep8
 
```

Comment:
I think this is much more readable than to have separate functions instead of the lambdas; I would like to suppress that warning in pep8.